### ISSUE TITLE
500 Error on webpagetest link

### ISSUE THREAD

**Issue by iAugur** [sergeychernyshev/showslow#114]
When clicking on the WebPageTest URL links on the home page you get a 500 error.
```
500 ShowSlow Error

Something went wrong. If it persists, please report it to issue tracker.

API call (http://www.webpagetest.org/getLocations.php?f=xml&k=<your_wpt_key>) failed:

<< go back to Show Slow
```

**Comment by sergeychernyshev** [comment]
Do you see any messages in error logs?

**Comment by iAugur** [comment]
Hi - It is on the main site http://www.showslow.com/

**Comment by sergeychernyshev** [comment]
OK, that explains it, let me check it out ;)

**Comment by sergeychernyshev** [comment]
Well, I fixed the 500 error (WPT API is served over HTTPS so I had to change the URL), but locations dropdown is broken, I have to dig a bit ddeper

**Comment by sergeychernyshev** [comment]
Fixed in 51eb2de0c066459a37617884e856914ef0251789 and b48f22ae80f943ba6e2c418e3a4f7514c98f8987
